tests/plugin-test/css-extract/cases/together-with-experiment-css/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const { CssExtractRspackPlugin } = require("@rspack/core");
+
+/** @type {import("@rspack/core").Configuration} */
+module.exports = {
```

Comment:
I don't understand what's the behavior when they use toghether, what's the difference between use them together and use experiments.css alone?

Reply:
I found a use case:

Rsbuild uses `CssExtractPlugin` by default. Sometimes user want to extend some custom CSS rules, then they can use `experiments.css` to handle specific CSS modules, it is easier to config than `CssExtractPlugin`.